Library/Homebrew/utils/git.rb

```diff
@@ -36,7 +36,7 @@ def last_revision_commit_of_files(repo, files, before_commit: nil)
     out, = Open3.capture3(
       HOMEBREW_SHIMS_PATH/"scm/git", "-C", repo, "log",
       "--pretty=format:%h", "--abbrev=7", "--max-count=1",
-      "--diff-filter=d", "--name-only", *args, "--", *files
+      "--diff-filter=d", "--name-only", *args, "--", T.unsafe(*files)
```

Comment:
@issyl0 Without `T.unsafe` we face the following error:
```
utils/git.rb:36: Splats are only supported where the size of the array is known statically https://srb.help/7019
    36 |    out, = Open3.capture3(
    37 |      HOMEBREW_SHIMS_PATH/"scm/git", "-C", repo, "log",
    38 |      "--pretty=format:%h", "--abbrev=7", "--max-count=1",
    39 |      "--diff-filter=d", "--name-only", *args, "--", *files
    40 |    )
```

Reply:
I played with this and you can get rid of the splat here and join the `files` array on spaces, which keeps Sorbet happy (and `brew extract` still works and passes the tests) without having to do `T.unsafe`:

```suggestion
      "--diff-filter=d", "--name-only", *args, "--", files.join(" ")
```